control-core.sh

```diff
@@ -0,0 +1,46 @@
+package: Control-Core
+version: "%(commit_hash)s"
+tag:  master
+build_requires:
+  - golang
+  - protobuf
+source: https://github.com/AliceO2Group/Control
+incremental_recipe: |
```

Comment:
Incremental recipe does the same as the base recipe (minus the `GOPATH` stuff, so how could it possibly work?), please drop it